src/package.json

```diff
@@ -10,16 +10,18 @@
   "scripts": {
     "prestart": "npm run build",
     "start": "node app.js",
-    "dev": "nodemon app.js",
-    "build": "node esbuild & npm run copy:root & npm run copy:webp & npm run copy:png & npm run build:svg",
+    "dev": "nodemon --ignore ./client/js/generated --exec 'npm run build:glean && node app.js'",
+    "build": "npm run build:glean && node esbuild & npm run copy:root & npm run copy:webp & npm run copy:png & npm run build:svg",
```

Comment:
Not sure if you want to add something like [**npm-run-all**](https://www.npmjs.com/package/npm-run-all) which might let you simplify to something like:

```diff
-"build": "npm run build:glean && node esbuild & npm run copy:root & npm run copy:webp & npm run copy:png & npm run build:svg",
+"build": "npm run build:glean && node esbuild & run-p copy:* build:svg",
```